kogito-editors-js/packages/boxed-expression-component/src/__tests__/components/FunctionExpression/FunctionExpression.test.tsx

```diff
@@ -78,12 +77,14 @@ describe("FunctionExpression tests", () => {
     await clearTableRow(container, baseElement);
 
     expect(mockedBroadcastDefinition).toHaveBeenLastCalledWith({
-      dataType: DataType.Undefined,
-      expression: expect.objectContaining({}),
+      dataType: undefined,
+      expression: {
+        logicType: LogicType.LiteralExpression,
+      },
       formalParameters: [],
       functionKind: "FEEL",
       logicType: "Function",
-      name: "p-1",
+      name: undefined,
```

Comment:
Why are we changing the expectations of this test case?

Reply:
Same as previous